content/providers/03-community-providers/10-portkey.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Portkey
+description: Portkey Provider for the AI SDK
+---
+
+# Portkey Provider
+
+[Portkey](https://portkey.ai/) is an AI Gateway that allows developers to connect to All the AI models in the world with a single API.
```

Comment:
does "All" need to be capitalised? Maybe we change the wording slightly to avoid confusion with AI SDK? something like, "allow developers to call any AI model through just one provider.